test/integration/login_flow_test.rb

```diff
@@ -28,10 +28,10 @@ class LoginFlowTest < ActionDispatch::IntegrationTest
       username: rusers(:jeff).username,
       password: 'secret'
     }
-    
+
     follow_redirect!
-    assert_response :redirect
-    assert_redirected_to  '/questions/new?tags=question%3Aquestion&template=question&title=What&redirect=question'
+    # assert_response :redirect
```

Comment:
OK - this is looking good, and we should be able to uncomment these again and they should work now! Great coding up above.